test/common/EnvVars.cpp

```diff
@@ -115,6 +228,18 @@ namespace RcclUnitTesting
     // Total number of reduction ops
     int numOps = ncclNumOps;
 
+    gpuPriorityOrder.resize(numDetectedGpus);
+    for(int i=0;i<numDetectedGpus;i++){
+      gpuPriorityOrder[i] = i;
+    }
+    if(isGfx94) {
+      bool cpxMode = false;
+      getDeviceMode(&cpxMode);
```

Comment:
The return value of `getDeviceMode` is never used, so if it can be simplified as I've suggested, you could rename it to `isDeviceCpxMode` and return `bool`.